modeller/chapter.md

```diff
@@ -135,18 +141,18 @@ if __name__ == "__main__":
 Before we dive into the `render` function, we should discuss a little bit of linear algebra.
 
 ### Coordinate Space
-For our purposes, a Coordinate Space is an origin point and a set of 3 basis vectors, usually the x, y, and z axes.
+For our purposes, a Coordinate Space is an origin point and a set of 3 basis vectors, usually the $x$, $y$, and $z$ axes.
 
 ### Point
-Any point in 3 dimensions can be represented as an offset in the x, y, and z directions from the origin point. The representation of a point is relative to the coordinate space that the point is in. The same point 
+Any point in 3 dimensions can be represented as an offset in the $x$, $y$, and $z$ directions from the origin point. The representation of a point is relative to the coordinate space that the point is in. The same point 
 has different representations in different coordinate spaces. Any point in 3 dimensions can be represented in any 3-dimensional coordinate space.
 
 ### Vector
-A vector is an x, y, and z value representing the difference between two points in the x, y and z axes, respectively.
+A vector is an $x$, $y$, and $z$ value representing the difference between two points in the $x$, $y$, and $z$ axes, respectively.
 
 ### Transformation Matrix
 In computer graphics, it is convenient to use multiple different coordinate spaces for different types of points. Transformation matrices convert points from one coordinate space to another coordinate space.
-To convert a vector `v` from one coordinate space to another, we multiply by a transformation matrix `M`: `v' = M v`.
+To convert a vector $v$ from one coordinate space to another, we multiply by a transformation matrix $M: v' = M v$.
```

Comment:
I changed this formatting to use MathJax notation -- can you make sure it still means what it's supposed to means? It can be compiled to HTML with the instruction at the start of the file, or if you like I can send a screen shot.